src/appsync-transformer.ts

```diff
@@ -274,23 +274,24 @@ export class AppSyncTransformer extends Construct {
 
     Object.keys(noneResolvers).forEach((resolverKey: any) => {
       const resolver = resolvers[resolverKey];
-
-      new Resolver(
-        this.nestedAppsyncStack,
-        `${resolver.typeName}-${resolver.fieldName}-resolver`,
-        {
-          api: this.appsyncAPI,
-          typeName: resolver.typeName,
-          fieldName: resolver.fieldName,
-          dataSource: noneDataSource,
-          requestMappingTemplate: MappingTemplate.fromFile(
-            resolver.requestMappingTemplate,
-          ),
-          responseMappingTemplate: MappingTemplate.fromFile(
-            resolver.responseMappingTemplate,
-          ),
-        },
-      );
+      if (resolver) {
```

Comment:
@kcwinner this is the spooky bit. It would never find `SubscriptionCreateCustomer` noneResolver. Not sure why it was even looking for that, to be honest. Tests all still passed?

Reply:
Looking into this bit... The subscriptions don't appear to get generated now

Reply:
After making the composite key change you *should* be able to remove this check. Or, to be safe, 
```
if (!resolver) throw new Error('...')
```